.packit.yaml

```diff
@@ -47,8 +47,10 @@ jobs:
       failure_comment:
         message: "Ephemeral COPR build failed. @containers/packit-build please check."
     targets: &fedora_copr_targets
-      - fedora-all-x86_64
-      - fedora-all-aarch64
+      - fedora-latest-stable-x86_64
+      - fedora-latest-stable-aarch64
+      - fedora-development-x86_64
+      - fedora-development-aarch64
```

Comment:
If the goal is to revert the `targets` changes in the file, a comment saying saying that we’d prefer to use `fedora-all-…` and that we are restricting F40 because of Go 1.22 would be nice (in both places).